packages/composables/src/internalHelpers/defaultApiParams.json

```diff
@@ -260,7 +260,10 @@
     }
   },
   "useCustomerOrders": {
-    "sort": "-createdAt",
+    "sort": [{
```

Comment:
@leichteckig thanks for adjustment <3 

here was the only issue regarding new format of Search Criteria, I mean the default json which is used, wasn't up to date with the new interface.